services/nublado2/values-idfprod.yaml

```diff
@@ -18,8 +18,10 @@ nublado2:
       PGPASSFILE: "/opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt"
       AWS_SHARED_CREDENTIALS_FILE: "/opt/lsst/software/jupyterlab/butler-secret/aws-credentials.ini"
       S3_ENDPOINT_URL: "https://storage.googleapis.com"
-      # technical debt: for needed for DP0 because deployment branch is not parametrized
-      AUTO_REPO_URLS: https://github.com/SimonKrughoff/system-test,https://github.com/rubin-dp0/tutorial-notebooks
+      # technical debt: for needed for DP0 because deployment branch is
+      # not parametrized
+      AUTO_REPO_SPECS: https://github.com/lsst-sqre/system-test@prod,https://github.com/rubin-dp0/tutorial-notebooks@prod
```

Comment:
I think I'd prefer this in the same order as in the other values files just for consistency

Reply:
Done.